### PR TITLE
Update availability-in-switzerland.md

### PR DESCRIPTION
**Pull request by Reeced40**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/5745
Added to note.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/country-and-region-availability-for-audio-conferencing-and-calling-plans/availability-in-switzerland.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ description: "Here are the Microsoft 365 and Office 365 services for Skype for B
 # Availability in Switzerland
 
 > [!NOTE]
-> You can get **SERVICE** numbers in Switzerland for use with conferencing, auto-attendant and call queues, but you cannot get **USER** numbers in Switzerland as calling plans are not available.
+> You can get **SERVICE** numbers in Switzerland for use with conferencing, auto-attendant and call queues, but you cannot get **USER** numbers in Switzerland as calling plans are not available. See also [PSTN Service Desk contact information](https://docs.microsoft.com/microsoftteams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk).
```
Comment (JohanFreelancer9) [inline Teams/country-and-region-availability-for-audio-conferencing-and-calling-plans/availability-in-switzerland.md:33]:
```suggestion
> You can get **SERVICE** numbers in Switzerland for use with conferencing, auto-attendant, and call queues, but you cannot get **USER** numbers in Switzerland as calling plans are not available. See also [PSTN Service Desk contact information](https://docs.microsoft.com/microsoftteams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk).
```

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by CarolynRowe** [comment]
@scanum this PR has unresolved conflicts.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by LanaChin** [comment]
@Reeced40, fyi that Calling Plans are now available in Switzerland. I updated these two Switzerland pages last week (which is likely why there's a conflict):

https://docs.microsoft.com/en-us/microsoftteams/country-and-region-availability-for-audio-conferencing-and-calling-plans/availability-in-switzerland
https://docs.microsoft.com/en-us/microsoftteams/manage-phone-numbers-for-your-organization/phone-number-management-for-switzerland


fyi @CarolynRowe

**Comment by Reeced40** [comment]
@scanum conflicts resolved thanks @CarolynRowe @LanaChin

**Comment by CarolynRowe** [comment]
It looks like there is a problem with this PR.  I am not seeing the changes.  I'm going to close.  Please reopen if there is still a change required.

**Pull request closed without merging**